Error with VARBert

```
INFO:libbs_vendored.jfx_bridge.bridge:serving! (jfx_bridge v1.0.0, Python 2.7.3)
INFO:libbs_vendored.jfx_bridge.bridge:Server launching in background - will continue to run after launch script finishes...
daila_plugin.py> Finished!
WARNING:libbs_vendored.jfx_bridge.bridge:Handling connection from ('127.0.0.1', 49171)
WARNING:libbs_vendored.jfx_bridge.bridge:Closing connection from ('127.0.0.1', 49171)
WARNING:libbs_vendored.jfx_bridge.bridge:Handling connection from ('127.0.0.1', 49172)
[INFO] | ghidra.interface | DAILA backend loaded! Initializing context menus now...
[INFO] | ghidra.interface | Predicting for function <Function: char4 * FUN_10140ea0(args=2); @0x140ea0 vars=4 len=0xb1>...
Traceback (most recent call last):
  File "C:\Users\idahe\OneDrive\Documents\Reverse Engineering\ghidra\ghidra_10.2_PUBLIC\Ghidra\Features\Base\ghidra_scripts\libbs_vendored\jfx_bridge\bridge.py", line 1365, in local_eval
    result = eval(eval_expr, eval_globals, eval_locals)
  File "<string>", line 1, in <module>
InvalidInputException: ghidra.util.exception.InvalidInputException: Parameter commit failed for function at 10140ea0
[ERROR] | ghidra.interface | Failed to evaluate remote code: [             HighFunctionDBUtil.updateDBVariable(sym, updates[0], updates[1], SourceType.ANALYSIS)             for sym, updates in symbols.items()         ]
[WARNING] | ghidra.interface | Exception in ctx menu callback varbert_rename_vars: 'Decompilation' object has no attribute 'replace'
[-] already enabled
[>] set current program: PkiPlugin.dll
[INFO] | ghidra.interface | Predicting for function <Function: char4 * FUN_10140ea0(args=2); @0x140ea0 vars=4 len=0xb1>...
Traceback (most recent call last):
  File "C:\Users\idahe\OneDrive\Documents\Reverse Engineering\ghidra\ghidra_10.2_PUBLIC\Ghidra\Features\Base\ghidra_scripts\libbs_vendored\jfx_bridge\bridge.py", line 1365, in local_eval
    result = eval(eval_expr, eval_globals, eval_locals)
  File "<string>", line 1, in <module>
InvalidInputException: ghidra.util.exception.InvalidInputException: Parameter commit failed for function at 10140ea0
[ERROR] | ghidra.interface | Failed to evaluate remote code: [             HighFunctionDBUtil.updateDBVariable(sym, updates[0], updates[1], SourceType.ANALYSIS)             for sym, updates in symbols.items()         ]
[WARNING] | ghidra.interface | Exception in ctx menu callback varbert_rename_vars_all: 'Decompilation' object has no attribute 'replace'
```

I get this error, and some weired variable names like `VARBERTpuVar2VARBERTvarid_siu0hiVARBERT` when trying to use VARBert.

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.8]
+        python-version: [3.10]
 
     steps:
       - uses: actions/checkout@v2

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.10]
+        python-version: [3.10.14]
 
     steps:
       - uses: actions/checkout@v2

--- a/dailalib/__init__.py
+++ b/dailalib/__init__.py
@@ -1,4 +1,4 @@
-__version__ = "3.4.1"
+__version__ = "3.4.2"
 
 from .api import AIAPI, LiteLLMAIAPI
 from libbs.api import DecompilerInterface

--- a/dailalib/api/litellm/prompts/prompt.py
+++ b/dailalib/api/litellm/prompts/prompt.py
@@ -1,5 +1,4 @@
 import json
-import typing
 import re
 from typing import Optional, Union, Dict, Callable
 import textwrap
@@ -48,7 +47,7 @@ class Prompt:
             raise Exception("api must be set before querying!")
 
         @AIAPI.requires_function
-        def _query_model(ai_api=self.ai_api, function=function, dec_text=dec_text, use_dec=use_dec) -> Union[Dict, str]:
+        def _query_model(ai_api=self.ai_api, function=function, dec_text=dec_text, **_kwargs) -> Union[Dict, str]:
             if not ai_api:
                 return {}
 

--- a/setup.cfg
+++ b/setup.cfg
@@ -5,7 +5,7 @@ url = https://github.com/mahaloz/DAILA
 classifiers =
     License :: OSI Approved :: BSD License
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.10
 license = BSD 2 Clause
 license_files = LICENSE
 description = The Decompiler Artificial Intelligence Language Assistant (DAILA) is a tool for adding AI to decompilers.
@@ -19,7 +19,7 @@ install_requires =
     Jinja2
     libbs>=1.18.1
 
-python_requires = >= 3.8
+python_requires = >= 3.10
 include_package_data = True
 packages = find:
 

--- a/setup.cfg
+++ b/setup.cfg
@@ -17,7 +17,7 @@ install_requires =
     litellm
     tiktoken
     Jinja2
-    libbs>=1.17.1
+    libbs>=1.18.1
 
 python_requires = >= 3.8
 include_package_data = True
@@ -29,4 +29,4 @@ console_scripts =
 
 [options.extras_require]
 full =
-    varbert>=2.1.0
+    varbert>=2.1.1

